### PR TITLE
feat(Footer): Load widget client only

### PR DESCRIPTION
**Pull request by montogeek**

### CODE REVIEW

File: app/components/prod.min.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+            .replace("[FONT_COLOR]", r)
+            .replace("[FONT_SIZE]", s)
+            .replace("[FONT_WEIGHT]", p)
+            .replace("[OUTLINE_COLOR]", i)
+            .replace("[COFFEE_COLOR]", l) + m
+        );
+      };
+    let a = document.querySelector('script[data-name="bmc-button"]');
+    if (a) {
+      a.nextElementSibling.innerHTML = bmcBtnWidget(
```
Comment (montogeek) [inline app/components/prod.min.js]: Original code uses `document.writeln` which is not longer recommend and also have a lot of security issues.
I just download it, update it to use good old `innerHTML` in a empty div :)

File: app/routes/_index.tsx [hunk 2]
```diff
@@ -88,12 +98,24 @@ export default function Index() {
...
+                data-slug="kirobyte"
+                data-color="#FFDD00"
+                data-emoji="🍻"
+                data-font="Cookie"
+                data-text="Buy me a beer"
+                data-outline-color="#000000"
+                data-font-color="#000000"
+                data-coffee-color="#ffffff"
+              ></script>
+              <div id="button" />
```
Comment (montogeek) [inline app/routes/_index.tsx:115]: This is where the button is inserted by `prod.min.js`

**Comment by montogeek** [comment]
Interesting, the component does not exist in the build, will investigate

**Comment by montogeek** [comment]
Locally it looks like this 
image

**Review by sescobb27 — APPROVED** [review]

**Pull request merged**